docs/gettingstarted/configuration.rst

```diff
@@ -6,9 +6,35 @@ Configuration
 We are fans of the `12factor <https://12factor.net/>`_ approach and portable code so you can configure most of Saleor using just environment variables.
 
 
+.. _payment_gateways_configuration:
+
 Payments Gateways
 -----------------
-For the configuration of the payment gateways, see :ref:`payment-gateways`.
+
+``CHECKOUT_PAYMENT_GATEWAYS``
+  This contains the list of enabled payment gateways, with the payment friendly name
+  to show to the user on the payment selection form.
+
+  For example, to add braintree to the enabled gateways,
+  you can do the following:
+
+  .. code-block:: python
+
+    CHECKOUT_PAYMENT_GATEWAYS = {
+        DUMMY: pgettext_lazy('Payment method name', 'Dummy gateway'),
+        BRAINTREE: pgettext_lazy('Payment method name', 'Brain tree')
+    }
+
+  The supported payment providers are:
+
+  - ``DUMMY`` (for tests purposes only!);
+  - ``BRAINTREE``;
+  - ``RAZORPAY``.
+
+
+``PAYMENT_GATEWAYS``
+    For information on how to configure payment gateways (API keys, miscellaneous information),
```

Comment:
How about ```(API keys, miscellaneous information and etc.)```?

Reply:
"and etc." would mean "and and so on" 😉

Reply:
I feel like `miscellaneous information` is already including the `etc.`. That sounds too much. But saying `(API keys, miscellaneous information, ...)` sound lighter. Let me know.

Reply:
+1 for ```(API keys, miscellaneous information, ...)``` 😄